types/workspace.ts

```diff
@@ -0,0 +1,23 @@
+import { URI } from './lsp'
+
+export const SELECT_WORKSPACE_ITEM_REQUEST_METHOD = 'aws/selectWorkspaceItem'
+export const OPEN_FILE_DIFF_NOTIFICATION_METHOD = 'aws/openFileDiff'
+
+export interface SelectWorkspaceItemParams {
+    canSelectFolders: boolean
```

Comment:
Have you checked with JB/Eclipse/VS to see if these options are more or less supported on their sides?